### ISSUE TITLE
Convert From Clipboard command fails

### ISSUE THREAD

**Issue by jtsom** [MariusAlch/vscode-json-to-ts#2]
Trying to use the Convert from Clipboard command, with valid JSON in the clipboard, fails and displays an error;
```
Command failed: cscript /Nologo C:\Users\John Tsombakos\.vscode-insiders\extensions\MariusAlchimavicius.json-to-ts-1.5.3\node_modules\copy-paste\platform\fallbacks\paste.vbs
```

This is on the latest VS Code Insiders on Windows 10 (and has happened pretty much since I installed the extension several weeks ago),

**Comment by LucasRTC** [comment]
I had the exactly same issue with the lastet version of VS Code on W10..

**Comment by naumov-dev** [comment]
I have the same issue
![image](https://user-images.githubusercontent.com/50377010/121886599-40836d00-cd1e-11eb-9564-7f0871128b35.png)

**Comment by xxKeefer** [comment]
i'm having issues using this in WSL2

**Comment by MariusAlch** [comment]
Should be resolved with version 1.8.0
